handbook/product/design/design-and-interaction-guidelines.md

```diff
@@ -0,0 +1,107 @@
+# Design and interaction guidelines
+
+Sourcegraph’s search experience is the application’s primary interaction. As such, the design is tailored to the search use case by prioritizing code over other elements.
+
+With the addition of campaigns, repository groups, and code insights, the product is quickly evolving. As such, its design must evolve to meet the experience requirements of these new features. 
+
+This document will describe the current design and interaction models and describe the principles we will utilize to inform our design and interaction choices moving forward.
+
+## Two design styles
+Sourcegraph utilizes two design and interaction styles which are tailored to the content they display:
+
+Search - An IDE like, full screen, high density, low contrast ui (for elements other than code) which centers on the display of code
+Web application - a more traditional web application style with larger margins and padding, increased contrast and standard web page design experience requirements
+
+## Code pages design
+
+In Sourcegraph’s search and file experiences, code is the most important element. The design of these pages reflects that priority. Contrast is balanced so that code stands out above all else. Panels clearly segment content and tools from the code. Pages are full width, and margins and padding are slim to increase the density of information which helps us display as much code on the screen as possible. 
```

Comment:
```suggestion
In Sourcegraph’s search, file, diff and other views, code is the most important element. To collectively refer to these pages we use the phrase 'code pages'. The design of these pages should focus on code as the most important element. Contrast is balanced so that code stands out above all else. Panels clearly segment content and tools from the code. Pages are full width, and margins and padding are slim to increase the density of information which helps us display as much code on the screen as possible. 
```